app/assets/stylesheets/components/_navbar.sass

```diff
@@ -0,0 +1,16 @@
+$Navbar-spacing-base: $Theme-spacing-base
+$Navbar-blue: Theme-color(color, header)
```

Comment:
Are you talking about the `Navbar-blue`, right? Because this isn't that modular. If the Navbar  text color change, the name will be wrong. Is that it? `$Navbar-text-color is a good name?`